guides/source/6_0_release_notes.md

```diff
@@ -641,10 +641,196 @@ Please refer to the [Changelog][active-support] for detailed changes.
+
+*   Add support for translations through I18n for `ActiveSupport::Inflector#ordinal`
+    and `ActiveSupport::Inflector#ordinalize`.
+    ([Pull Request](https://github.com/rails/rails/pull/32168))
+
+*   Add `before?` and `after?` methods to `Date`, `DateTime`,
+    `Time`, and `TimeWithZone`.
+    ([Pull Request](https://github.com/rails/rails/pull/32185))
+
+*   Fix bug where `URI.unscape` would fail with mixed Unicode/escaped character
+    input.
+    ([Pull Request](https://github.com/rails/rails/pull/32183))
+
+*   Fix bug where `ActiveSupport::Cache` will massively inflate the storage
+    size when compression is enabled.
+    ([Pull Request](https://github.com/rails/rails/pull/32539))
+
+*   Redis cache store: `delete_matched` no longer blocks the Redis server.
+    ([Pull Request](https://github.com/rails/rails/pull/32614))
+
+*   Fix bug where `ActiveSupport::TimeZone.all` would fail when tzinfo data for
+    any timezone defined in `ActiveSupport::TimeZone::MAPPING` is missing.
+    ([Pull Request](https://github.com/rails/rails/pull/32613))
+
+*   Add `Enumerable#index_with` which allows creating a hash from an enumerable
+    with the value from a passed block or a default argument.
+    ([Pull Request](https://github.com/rails/rails/pull/32523))
+
+*   Allow `Range#===` and `Range#cover?` methods to work with `Range` argument.
+    ([Pull Request](https://github.com/rails/rails/pull/32938))
+
+*   Support key expiry in `increment/decrement` operations of RedisCacheStore.
+    ([Pull Request](https://github.com/rails/rails/pull/33254))
+
+*   Add cpu time, idle time, and allocations features to log subscriber events.
+    ([Pull Request](https://github.com/rails/rails/pull/33449))
+
+*   Add support for event object to the Active Support notification system.
+    ([Pull Request](https://github.com/rails/rails/pull/33451))
+
+*   Add support for not caching `nil` entries by introducing new option `skip_nil`
+    for `ActiveSupport::Cache#fetch`.
+    ([Pull Request](https://github.com/rails/rails/pull/25437))
+
+*   Add `Array#extract!` method which removes and returns the elements for which
+    block returns a true value.
+    ([Pull Request](https://github.com/rails/rails/pull/33137))
+
+*   Keep an HTML safe string HTML safe after slicing.
+    ([Pull Request](https://github.com/rails/rails/pull/33808))
+
+*   Add support for tracing constant autoloads via logging.
+    ([Commit](https://github.com/rails/rails/commit/c03bba4f1f03bad7dc034af555b7f2b329cf76f5))
+
+*   Define `unfreeze_time` as an alias of `travel_back`.
+    ([Pull Request](https://github.com/rails/rails/pull/33813))
+
+*   Changed `ActiveSupport::TaggedLogging.new` to return a new logger instance
+    instead of mutating the one received as argument.
+    ([Pull Request](https://github.com/rails/rails/pull/27792))
+
+*   Treat `#delete_prefix`, `#delete_suffix` and `#unicode_normalize` methods
+    as non HTML safe methods.
```

Comment:
Based on the comments by @connorshea, this is just a consistency recommendation from his recommendations:
```suggestion
    as non HTML-safe methods.
```